enchanted-swiftie/cogs/songs.py

```diff
@@ -0,0 +1,66 @@
+import billboard
+import datetime as dt
+from discord.ext import commands
+
+
+class Songs(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+        self.singles_dict = {}
+        self.last_checked = None
+
+    @commands.command(aliases=["Singles"])
+    async def singles(self, ctx):
+        """Returns all of Taylor's singles that peaked in the hot 100"""
+        # Goes over all the charts up to current date
+        while self.last_checked + dt.timedelta(7) < dt.datetime.date(dt.datetime.now()):
+            # Creates a chart list for the week
+            chart = billboard.ChartData("hot-100", str(self.last_checked))
+            self.add_singles_to_dict(chart)
+            # Goes over to the next chart
+            self.last_checked += dt.timedelta(7)
+
+        message = self.format_message()
+        with open(r"support files\billboard.txt", "w") as file:
```

Comment:
file is an existing name
```suggestion
        with open(r"support files\billboard.txt", "w") as singles_file:
```